### PR TITLE
add an example request for ISBN form & update babel and eslint related packages

### PR DESCRIPTION
**Pull request by mvillivald**
1. Added an example request for ISBN form, console.logging the data coming from the form.
2. Updated babel and eslint related packages to the latest versions. This only applies to devDependencies, does not affect the user interface in any way. The reason is the earlier update of the eslint package to version 8.8.0.
3. Fix several eslint related "issues" that appeared after updating libraries - mostly spaces, commas, and so on.

### CODE REVIEW

File: src/frontend/components/form/isbnIsmnRegistrationForm/content.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ export function getFormPages(intl) {
 					options: [
```
Comment (aatuny) [inline src/frontend/components/form/isbnIsmnRegistrationForm/content.js:45]: Lint rule regarding comma-dangle should be turned off for now.

File: src/frontend/components/form/isbnIsmnRegistrationForm/preview.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-closing-tag-location */
```
Comment (aatuny) [inline src/frontend/components/form/isbnIsmnRegistrationForm/preview.js:1]: Please provide reasoning for disabling this lint-rule.

**Review by aatuny — CHANGES_REQUESTED** [review]

File: src/frontend/components/form/isbnIsmnRegistrationForm/content.js (hunk 1 above)
Comment (mvillivald) [reply src/frontend/components/form/isbnIsmnRegistrationForm/content.js]: Fixed

File: src/frontend/components/form/isbnIsmnRegistrationForm/preview.js (hunk 2 above)
Comment (mvillivald) [reply src/frontend/components/form/isbnIsmnRegistrationForm/preview.js]: Fixed

**Review by aatuny — CHANGES_REQUESTED** [review]
Remove code that has been commented out from preview.js if there's no identified use for it.

**Review by aatuny — APPROVED** [review]

**Pull request merged**